crates/biome_cli/src/execute/migrate/prettier.rs

```diff
@@ -521,7 +521,7 @@ mod tests {
   "semi": true,
   "singleQuote": true,
   "tabWidth": 2,
-  "trailingComma": "es5",
+  "trailingCommas": "es5",
```

Comment:
We shouldn't change this because the option is named `trailingComma` in the prettier configuration: https://prettier.io/docs/en/options.html#trailing-commas

(Although, it uses the plural form in their document heading, which makes it further confusing)